packages/exoflex/src/components/Menu.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Menu as PaperMenu } from 'react-native-paper';
 import Text from './Text';
 import useTheme from '../helpers/useTheme';
+import { AccessibilityProps, View } from 'react-native';
 
 export type MenuProps = OmitPaperTheme<typeof PaperMenu>;
```

Comment:
What about accessibility for Menu?

Reply:
I skipped it as we can't access inside Paper's Menu. But should we just wrap it in a View and put the a11y props into that View similar to what I did with the Menu Item?

Reply:
Yes, let's wrap the Menu inside a View, similar to Menu Item.

Reply:
Actually, it kinda feels off by doing that as the component that we wrapped turns out to be the `anchor` instead of the parent of `Menu.Item`.

Here is the wrapper that we give a role as `menu`.
![image](https://user-images.githubusercontent.com/4923122/80074825-f799b380-8573-11ea-80dc-b930544ebc9a.png)

From the image above, the wrapper (menupopup) is only wrapping the `anchor` which is a button. That's why it doesn't show any menu item from the tree.

And here is the menu item.
![image](https://user-images.githubusercontent.com/4923122/80075074-519a7900-8574-11ea-8559-858fd06068f1.png)

Reply:
And this is what the tree looks like with Radio.
![image](https://user-images.githubusercontent.com/4923122/80075363-c66db300-8574-11ea-9841-763f0957f55a.png)

Reply:
I see. In that case, I'm indifferent on whether we add the `menu` accessibility at all.

Reply:
Alright then, I think we could just skip it until Paper implement it or we fork Paper and implement it from there.